plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/editors/data/preferences/PrefPageResultSetPresentationGrid.java

```diff
@@ -52,7 +52,7 @@
     private Button showBooleanAsCheckbox;
     private Combo gridDoubleClickBehavior;
     private Text gridRowBatchSize;
-    private Text maxDefColumnWidth;
+    private Spinner maxDefColumnWidth;
```

Comment:
Please avoid using spinner, it looks goofy on Linux OSes. (see also 547b2909f201aef24e7b095592297edd7973845a)

You can use custom `VerifyListener` applied on the `Text` widget using `addVerifyListener` that will check the bounds.

Reply:
Perhaps it may be OK to use `Spinner` here.